src/ParseFile.js

```diff
@@ -134,10 +134,16 @@ export default class ParseFile {
    * Gets the url of the file. It is only available after you save the file or
    * after you get the file from a Parse.Object.
    * @method url
+   * @param {Object} options An object to specify url options
    * @return {String}
    */
-  url(): ?string {
-    return this._url;
+  url(options?: { secure?: boolean }): ?string {
+    options = options || {};
+    if (options.secure) {
```

Comment:
`if (this._url && options.forceSecure)`

If the file doesn't have a URL set, this would have previously returned undefined; now, it'll crash.